tests/end_to_end/test_vulnerability_detector/cases/test_vulnerability.yaml

```diff
@@ -1,483 +1,298 @@
+          amd64: luxon-3.0.0
+          arm64v8: luxon-3.0.0
+  teardown:
+    operation: remove_package
+    package:
+      centos:
+        amd64: grafana-10.0.0-1
+        arm64v8: grafana-10.0.0-1
+      ubuntu:
+        amd64: grafana-10.0.0
+        arm64v8: grafana-10.0.0
+      windows:
+        amd64: node-v20.5.1
+      macos:
+        amd64: luxon-3.0.0
+        arm64v8: luxon-3.0.0
 
 - case: Installation of a non vulnerable package
   id: install_package_non_vulnerable
   description: |
     Installation of a non vulnerable package
-      macos:
-        Used Package: http-proxy 0.7.0 - npm Format
-        "CVE": [],
-      windows:
-        Used Package: Node 18.20.0 - .msi Format
-        "CVE": [],
-      ubuntu:
-        Used Packages: Grafana 9.5.13 - .deb Format
-        CVE: [],
-      centos:
-        Used Packages: Grafana 9.5.13 - .rpm Format
-        CVE: [],
-  preconditions: null
   body:
-    tasks:
-      - operation: install_package
-        target: agent
-        check:
-          alerts: true
-          state_index: true
-        package:
-          centos:
-            amd64: grafana-9.5.13-1
-            arm64v8: grafana-9.5.13-1
-          ubuntu:
-            amd64: grafana-9.5.13
-            arm64v8: grafana-9.5.13
-          windows:
-            amd64: node-v18.20.0
-          macos:
-            amd64: http-proxy-0.7.0
-            arm64v8: http-proxy-0.7.0
+    operation: install_package
+    package:
+      centos:
+        amd64: grafana-9.5.13-1
+        arm64v8: grafana-9.5.13-1
+      ubuntu:
+        amd64: grafana-9.5.13
+        arm64v8: grafana-9.5.13
+      windows:
+        amd64: node-v19.5.0
+      macos:
+        amd64: http-proxy-0.7.0
+        arm64v8: http-proxy-0.7.0
+
 - case: 'Remove: Non vulnerable package'
   id: remove_non_vulnerable_packge
   description: |
     Removal of a non vulnerable package
-      macos:
-        Used Package: http-proxy 0.7.0 - npm Format
-        "CVE": [],
-      windows:
-        Used Package: Node 18.20.0 - .msi Format
-        "CVE": [],
-      ubuntu:
-        Used Packages: Grafana 9.5.13 - .deb Format
-        CVE: []
-      centos:
-        Used Packages: Grafana 9.5.13 - .rpm Format
-        CVE: [],
   body:
-    tasks:
-      - operation: remove_package
-        target: agent
-        check:
-          alerts: true
-          state_index: true
-        package:
-          centos:
-            amd64: grafana-9.5.13-1
-            arm64v8: grafana-9.5.13-1
-          ubuntu:
-            amd64: grafana-9.5.13
-            arm64v8: grafana-9.5.13
-          windows:
-            amd64: node-v18.20.0
-          macos:
-            amd64: http-proxy-0.7.0
-            arm64v8: http-proxy-0.7.0
+    operation: remove_package
+    package:
+      centos:
+        amd64: grafana-9.5.13-1
+        arm64v8: grafana-9.5.13-1
+      ubuntu:
+        amd64: grafana-9.5.13
+        arm64v8: grafana-9.5.13
+      windows:
+        amd64: node-v19.5.0
```

Comment:
Why have the https://github.com/wazuh/wazuh-qa/pull/5234 changes been reverted?

Reply:
Done eb2900021a5d9a3a212fa9c629987bf607bb37b5